src/components/shared/Datatable.vue

```diff
@@ -177,7 +177,8 @@
             vuetableConfig: {
                 deep: true,
                 handler: debounce(function () {
-                    if (this.$refs.vuetable && this.$refs.vuetable.$el) this.$refs.vuetable.refresh()
+                    if (!this.$refs.vuetable || (this.$refs.vuetable && !this.$refs.vuetable.$el)) return
```

Comment:
Isn't the second check for `this.$refs.vuetable` redundant? If it was falsey, that would satisfy the left hand side of the statement and so it would only ever test the right hand side if it had a truthy value? Unless I am missing something?

```js
if (!this.$refs.vuetable || !this.$refs.vuetable.$el) return
```